gravityflow/gravityflow
# Fix Workflow Step Schedule Expiration by Date Field

## Description
Workflow Expiration Setting by 'Date Field' is not well aligned with Gravity Forms 2.5. This PR fixes the CSS to make all the dropdowns for the expiration settings appear on a single line.

This PR fixes [gravityflow/backlog#135](https://github.com/gravityflow/backlog/issues/135).

## Testing instructions
1. Create a form with a date field.
2. Add a Workflow step and enable the expiration setting
3. Select the date field (as created in step 1) as the expiration value.
4. Notice the display for the expration settings appears misformatted and on separate lines.
5. Switch to this branch and repeat steps 3 and 4, confirm the appearance now.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
